packages/fxa-content-server/app/tests/spec/lib/survey-targeter.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import SurveyTargeter from 'lib/survey-targeter';
+import { assert } from 'chai';
+
+describe('lib/SurveyTargeter', () => {
+  let surveyTargeter;
+
+  beforeEach(() => {
+    surveyTargeter = new SurveyTargeter({
+      surveys: [{
+        id: 'portugese-speaking-mobile-users-in-southern-hemisphere',
+        conditions: {},
+        view: 'settings',
+        rate: 0.1,
+        url: 'https://www.surveygizmo.com/s3/5541940/pizza',
+      }]
+    });
+  });
+
+  it('surveyTargeter declares a surveyMap on initialization', () => {
+    assert.isDefined(surveyTargeter.surveyMap);
```

Comment:
I'll be changing how and where that map is built in another PR so we don't need to make changes here.  But this test is passing simply from line 6 in `survey-targeter` where `surveyMap` is set to an empty object.  It does not test anything else about it.